elixir-mode.el

```diff
@@ -167,7 +167,7 @@
                          (or "_" "__MODULE__" "__DIR__" "__ENV__" "__CALLER__"
                              "__block__" "__aliases__")
                          symbol-end))
-      (sigils . ,(rx "~" (or "B" "C" "R" "S" "b" "c" "r" "s" "w")))))
+      (sigils . ,(rx "~" (or "D" "B" "C" "R" "S" "b" "c" "r" "s" "w")))))
```

Comment:
Should `"D"` be after `"C"` ? Because original order is ascii(alphabetical) order.